website/src/documentation/sdks/euphoria.md

```diff
@@ -104,46 +97,31 @@ pipeline.run();
 
 Euphoria API is composed from a set of operators, which allows you to construct `Pipeline` according to your application needs.
 
-### Datasets
-Euphoria uses the concept of 'Datasets' to describe data pipeline between `Operators`. This concept is similar to Beam's `PCollection` and can be converted back and forth through:
```

Comment:
Should we mention Beam's `PCollection` here or do we just assume users are familiar with it?

Reply:
Right now we are assuming that user is familiar with the basic beam model, which has great documentation. We may reconsider this, if we see new users struggling.